MAUI/TreeMap/migration.md

```diff
@@ -352,27 +358,27 @@ public class PopulationViewModel
 </tr>
 <tr>
 <td>Nil</td>
-<td>Item</td>
+<td>{{'[Item](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TreeMap.TreeMapItemInfo.html#Syncfusion_Maui_TreeMap_TreeMapItemInfo_Item)'| markdownify }}</td>
 <td>Gets or sets the item associated with the TreeMapItemInfo.</td>
 </tr>
 <tr>
 <td>Nil</td>
-<td>GroupLevel</td>
+<td>{{'[GroupLevel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TreeMap.TreeMapItemInfo.html#Syncfusion_Maui_TreeMap_TreeMapItemInfo_GroupLevel)'| markdownify }}</td>
 <td>Gets or sets the group level of the item.</td>
 </tr>
 <tr>
 <td>Nil</td>
-<td>Background</td>
+<td>{{'[Background](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TreeMap.TreeMapItemInfo.html#Syncfusion_Maui_TreeMap_TreeMapItemInfo_Background)'| markdownify }}</td>
 <td>Gets the background brush for the item.</td>
 </tr>
 <tr>
 <td>Nil</td>
-<td>PrimaryValueText</td>
+<td>{{'[PrimaryValueText](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TreeMap.TreeMapItemInfo.html#Syncfusion_Maui_TreeMap_TreeMapItemInfo_PrimaryValueText)'| markdownify }}</td>
 <td>Gets the primary value text of the item.</td>
 </tr>
 </table>
 
-### LeafItemSettings
+### TreeMapLeafItemSettings
```

Comment:
Remove tree map and  Rename to `LeafItemSettings.`

Update same in all related places.